2022-06-21 19:59:11,314 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2022-06-21 19:59:11,317 INFO supervisord started with pid 8
2022-06-21 19:59:12,322 INFO spawned: 'nginx' with pid 11
2022-06-21 19:59:12,327 INFO spawned: 'app' with pid 12
2022-06-21 19:59:13,325 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-06-21 19:59:13,325 INFO success: app entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
Bottle v0.12.21 server starting up (using WSGIRefServer())...
Listening on http://localhost:5000/
Hit Ctrl-C to quit.

[CTRL] POST /setup
[Time: 21.06.22 19:59:17] [Level: info] id: db951abba81be53a
[Time: 21.06.22 19:59:17] [Level: info] coordinator: True
[Time: 21.06.22 19:59:17] [Level: info] clients: ['da33a2b3636a2438', 'db951abba81be53a']
[Time: 21.06.22 19:59:17] [Level: info] state: initial
[Time: 21.06.22 19:59:17] [Level: info] transition: read
127.0.0.1 - - [21/Jun/2022 19:59:17] "POST /api/setup HTTP/1.0" 200 0
[CTRL] GET /status
127.0.0.1 - - [21/Jun/2022 19:59:17] "GET /api/status HTTP/1.0" 200 126
[Time: 21.06.22 19:59:18] [Level: info] state: read
[Time: 21.06.22 19:59:18] [Level: info] transition: aggregate
[Time: 21.06.22 19:59:19] [Level: info] state: aggregate
[CTRL] GET /status
127.0.0.1 - - [21/Jun/2022 19:59:20] "GET /api/status HTTP/1.0" 200 131
[CTRL] POST /data
127.0.0.1 - - [21/Jun/2022 19:59:22] "POST /api/data?client=da33a2b3636a2438 HTTP/1.0" 200 0
[Time: 21.06.22 19:59:22] [Level: info] transition: await
[CTRL] GET /status
127.0.0.1 - - [21/Jun/2022 19:59:23] "GET /api/status HTTP/1.0" 200 126
[CTRL] GET /data
127.0.0.1 - - [21/Jun/2022 19:59:23] "GET /api/data HTTP/1.0" 200 44
[Time: 21.06.22 19:59:23] [Level: info] state: await
[Time: 21.06.22 19:59:23] [Level: info] Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/FeatureCloud/app/engine/app.py", line 169, in guarded_run
    self.run()
  File "/usr/local/lib/python3.8/site-packages/FeatureCloud/app/engine/app.py", line 184, in run
    transition = self.current_state.run()
  File "/app/states.py", line 86, in run
    c.delete_columns(to_drop)
  File "/app/states.py", line 25, in delete_columns
    self.df.drop(columns=to_drop)
  File "/usr/local/lib/python3.8/site-packages/pandas/util/_decorators.py", line 311, in wrapper
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/pandas/core/frame.py", line 4954, in drop
    return super().drop(
  File "/usr/local/lib/python3.8/site-packages/pandas/core/generic.py", line 4267, in drop
    obj = obj._drop_axis(labels, axis, level=level, errors=errors)
  File "/usr/local/lib/python3.8/site-packages/pandas/core/generic.py", line 4311, in _drop_axis
    new_axis = axis.drop(labels, errors=errors)
  File "/usr/local/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 6644, in drop
    raise KeyError(f"{list(labels[mask])} not found in axis")
KeyError: "[('msp_0001',), ('msp_0001',)] not found in axis"

